src/devices/Lp55231/MiscFlags.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace Iot.Device.Lp55231
+{
+    /// <summary>
+    /// Flags related to the REG_MISC register
+    /// </summary>
+    [Flags]
+    public enum MiscFlags : byte
+    {
+        /// <summary>
+        /// Use internal clock
+        /// </summary>
+        ClockSourceSelection = 0b00000001,
```

Comment:
tip: it's better to make it 0b0000_0001
so using the underscore to separate in 2 the binary representation. The eye have hard time to catch more than groups of 3 or 4 elements. And if something is missing on a 8 elements in a raw, it definitely won't on a 2 time 4, you're increasing the changes :-)

Reply:
Done